django/core/files/utils.py

```diff
@@ -24,6 +27,8 @@ def __init__(self, file):
     write = property(lambda self: self.file.write)
     writelines = property(lambda self: self.file.writelines)
     xreadlines = property(lambda self: self.file.xreadlines)
+    if six.PY3:
```

Comment:
That is how it was originally written, but we decided explicitness was better. I don't have a preference here myself.